chapters/connectors.tex

```diff
@@ -107,9 +107,14 @@ \subsection{Expandable Connectors}\label{expandable-connectors}
 This applies to both variables of simple types, and variables of structured types.
 
 Then connections containing expandable connectors are elaborated:
-\begin{itemize}
+\begin{enumerate}
+\item
+  If a connect-equation references a potentially present component \emph{as part of the argument} it will be marked as being present, which will allowing a connection to an undeclared connector inside it.
```

Comment:
Typo:
```suggestion
  If a connect-equation references a potentially present component \emph{as part of the argument} it will be marked as being present, which will allow a connection to an undeclared connector inside it.
```